.github/workflows/build.yml

```diff
@@ -13,8 +13,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        subset: [network, consensus]
         ghc: ["8.6.5", "8.10.2"]
         os: [ubuntu-latest, windows-latest]
+        exclude:
+          - os: windows-latest
+            subset: consensus
```

Comment:
Does it fail to build? If it doesn't, I would enable it.

Reply:
I would like to submit that as a separate PR.  Some further work is needed to set up `libsodium` for the Windows build.

Reply:
I ended up fixing the Windows build.